### PR TITLE
add support for another mountpoint than /mnt

### PR DESCRIPTION
**Pull request by Lassulus**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -1,6 +1,8 @@
-{ lib ? import <nixpkgs/lib> }:
+{ lib ? import <nixpkgs/lib>
+, rootMountPoint ? "/mnt"
```
Comment (Mic92) [inline default.nix:2]: Should the cli not also expose this?

Comment (Lassulus) [reply default.nix:2]: ah yeah, haven't used the cli in a while :D probably should also do some tests with it

**Comment by Mic92** [comment]
bors merge

File: types.nix [hunk 2]
```diff
@@ -223,7 +223,7 @@ rec {
     */
     zapCreateMount = devices: ''
       set -efux
-      umount -Rv /mnt || :
+      umount -Rv ${rootMountPoint} mnt || :
```
Comment (Mic92) [inline types.nix:226]: Shoud this not be quoted?

Comment (Mic92) [inline types.nix:226]:
```suggestion
      umount -Rv "${rootMountPoint}" mnt || :
```

**Comment by Mic92** [comment]
bors merge

**Pull request merged**